src/components/column.svelte

```diff
@@ -27,16 +29,11 @@
   }}
   on:dropped
 >
-  <header
-    class="text-sm font-thin h-5 border-b-2 border-blue-600 overflow-hidden cursor-text"
-    on:click={() => (isTitleEditing = true)}
+
+
+  <input on:mousedown|stopPropagation on:blur="{(e) => editColumnName(id,e.target.value)}" value={title} type="text" 
```

Comment:
Stops handling mousedown event from input element so it wont be passed to the drag handler